src/PharoLauncher-CLI-Tests/PhLVMCliCommandTest.class.st

```diff
@@ -0,0 +1,54 @@
+Class {
+	#name : #PhLVMCliCommandTest,
+	#superclass : #TestCase,
+	#instVars : [
+		'context'
+	],
+	#category : #'PharoLauncher-CLI-Tests'
+}
+
+{ #category : #instance }
+PhLVMCliCommandTest >> PhLCliSpecificCommand [
+	^ SubclassResponsibility 
+]
+
+{ #category : #adding }
+PhLVMCliCommandTest >> addVmNamed: anImageName at: someDictionaryIndice [
+	^ self vmRepository addVmNamed: anImageName at: someDictionaryIndice 
```

Comment:
I do not think it is needed to specify the key where the VM will be store. You could simply use the vm id.
Then the code above will be:
```smalltalk
^ self vmRepository addVmNamed: anImageName
```